lmms_eval/tasks/docvqa_hu/utils.py

```diff
@@ -18,6 +18,13 @@ def docvqa_doc_to_text(doc, model_specific_prompt_kwargs):
     return f"{pre_prompt}{question}{post_prompt}"
 
 
+def docvqa_doc_to_target(doc):
+    answer = doc['Short Answer']
```

Comment:
For hungarian docvqa, will `Short Answer` always exist as None? Wondering if we should do .get('Short Answer', None) if not.

Reply:
Short Answer is valid for all but a few docs, this is just to catch those few cases. For a dataset with only 54 datapoints, a few can make a noticeable dent in accuracy.